### PR TITLE
Prof crd mismatch

### PR DESCRIPTION
**Pull request by kunmingg**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/156)

### CODE REVIEW

File: profiles/base/crd.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
   names:
     kind: Profile
     plural: profiles
-  scope: Namespaced
+  scope: Cluster
```
Comment (hougangliu) [inline profiles/base/crd.yaml]: about this change, we had discussed in https://github.com/kubeflow/manifests/pull/29#discussion_r277858777
@kkasravi WDYT?

Comment (kunmingg) [reply profiles/base/crd.yaml]: Latest CRD on controller side is Cluster scope:
https://github.com/kubeflow/kubeflow/blob/master/components/profile-controller/config/crds/kubeflow_v1alpha1_profile.yaml

**Pull request merged**